bin/bootstrax

```diff
@@ -176,7 +168,7 @@ timeouts = {
 # hence, we can check if there is sufficient diskspace and if not, wait a while.
 # Below are the max number of times and number of seconds bootstrax will wait.
 wait_diskspace_max_space_percent = 90
-wait_diskspace_n_max = 100  # times
+wait_diskspace_n_max = 60 * 24 * 7 # times
```

Comment:
An entire week?

Reply:
Well, it doesn't do anything until it simply dies. It just waits until you free up some space. We could even set it to infinite.